config/locales/en/providers.yml

```diff
@@ -1839,4 +1847,4 @@ en:
           option_bullets:
             - make a new application for your client if you still need to submit one
             - go back to your applications
-          applications: Make a new application
+          applications: Make a new application
```

Comment:
@skdejong just need to remove this added blank line

Reply:
@kmahern, I can't see a blank line in here?

Reply:
@kmahern, I can't see the blank line in the file that needs to be deleted.

Reply:
@skdejong apologies I mean I think the final empty line was accidentally deleted. Can we add it back?